README.md

```diff
@@ -9,14 +9,97 @@ Step 2 : Open project in Android Studio
+
+
+![Screenshot_20191216-210955](https://user-images.githubusercontent.com/45551012/70924233-314f3080-204f-11ea-9240-e6213b6f1af7.png)
+
+
+![Screenshot_20191216-211104](https://user-images.githubusercontent.com/45551012/70924279-40ce7980-204f-11ea-91db-683412b87847.png)
+
+
+![Screenshot_20191216-211142](https://user-images.githubusercontent.com/45551012/70924310-4cba3b80-204f-11ea-8071-7e1f2fe0c6cf.png)
+
+
+![Screenshot_20191216-211147](https://user-images.githubusercontent.com/45551012/70924341-593e9400-204f-11ea-94c4-3516b09fb92e.png)
+
+
+![Screenshot_20191216-211204](https://user-images.githubusercontent.com/45551012/70924377-69ef0a00-204f-11ea-9b76-6b7020d8000b.png)
+
+
+![Screenshot_20191216-211209](https://user-images.githubusercontent.com/45551012/70924409-770bf900-204f-11ea-8807-482af429e470.png)
+
+
+![Screenshot_20191216-211302](https://user-images.githubusercontent.com/45551012/70924451-855a1500-204f-11ea-8a34-b92656843fbf.png)
+
+
+![Screenshot_20191216-211351](https://user-images.githubusercontent.com/45551012/70924491-99057b80-204f-11ea-8f33-0b00b439d0a8.png)
+
+
+![Screenshot_20191216-211356](https://user-images.githubusercontent.com/45551012/70924510-a3277a00-204f-11ea-97c2-061f1d5026d4.png)
+
+
+![Screenshot_20191216-211407](https://user-images.githubusercontent.com/45551012/70924529-af133c00-204f-11ea-8370-637fd2aea506.png)
+
+
+![Screenshot_20191216-211424](https://user-images.githubusercontent.com/45551012/70924576-be928500-204f-11ea-9ce6-4c3e01858359.png)
+
+
+![Screenshot_20191216-211603](https://user-images.githubusercontent.com/45551012/70924601-cbaf7400-204f-11ea-8bc5-da7a65d58ebb.png)
+
+
+![Screenshot_20191216-211614](https://user-images.githubusercontent.com/45551012/70924624-d79b3600-204f-11ea-9436-330745b55891.png)
+
+
+![Screenshot_20191216-211622](https://user-images.githubusercontent.com/45551012/70924647-e41f8e80-204f-11ea-8d58-97eb5f0089cf.png)
+
+
+![Screenshot_20191216-211633](https://user-images.githubusercontent.com/45551012/70924674-eeda2380-204f-11ea-888c-187916ec4b9e.png)
+
+
+![Screenshot_20191216-211639](https://user-images.githubusercontent.com/45551012/70924695-f8fc2200-204f-11ea-90f2-5ae67196027c.png)
+
+
+![Screenshot_20191216-211717](https://user-images.githubusercontent.com/45551012/70924710-05807a80-2050-11ea-8cb7-334f4e421d5e.png)
+
+
+![Screenshot_20191216-211732](https://user-images.githubusercontent.com/45551012/70924731-0fa27900-2050-11ea-99c1-30630e100d16.png)
+
+
+![Screenshot_20191216-211815](https://user-images.githubusercontent.com/45551012/70924765-19c47780-2050-11ea-8f4f-26777ae34a2e.png)
+
+
+![Screenshot_20191216-211848](https://user-images.githubusercontent.com/45551012/70924776-234ddf80-2050-11ea-8fd8-9e34390c7bad.png)
+
+
+![Screenshot_20191216-211928](https://user-images.githubusercontent.com/45551012/70924805-2ea10b00-2050-11ea-8562-5087a2f5b136.png)
+
```

Comment:
@amit6663  change the image sizes . it's looking quite big . make it look small and concised .